to_nwb/neuroscope.py

```diff
@@ -6,25 +6,26 @@
 
 import numpy as np
 import pandas as pd
-from bs4 import BeautifulSoup
+from pathlib import Path
+from lxml import etree as et
 from pynwb.behavior import SpatialSeries
 from pynwb.ecephys import ElectricalSeries, LFP, SpikeEventSeries, EventWaveform#, UnitSeries
 from hdmf.backends.hdf5.h5_utils import H5DataIO
 from hdmf.data_utils import DataChunkIterator
 from pynwb.misc import AnnotationSeries
 from tqdm import tqdm
+from typing import Union
 
 from .utils import check_module
 
+PathType = Union[str, Path, None]
```

Comment:
A better way to allow for `None` is to define this as 

```python
Path = Union[str, Path]
```

and then when you want to allow `None`, use 

```python
from typing import Optional
def my_func(opt_str: Optional[Path])
    pass
```

Reply:
Good to know! The `Path = Union[str, Path, None]` is what we currently use in `SpikeInterface` so if we change it here we can also change it over there.

That said, I think I might save all the `Path` usage for a different branch/PR and just leave this one for the `lxml` changes, but I'll definitely use that trick whenever I can.